modules/account/controllers/profile.classic.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+* @package   test
+* @subpackage account
+* @author    test
+* @copyright 2019 Laurent Jouanneau and other contributors
+* @link      
+* @license    All rights reserved
+*/
+
+use Jelix\Authentication\Account\Manager;
+use Jelix\Authentication\Account\Account;
+
+class profileCtrl extends jController {
+
+    function index() {
+        $rep = $this->getResponse('html');
+
+        $currentUser = Manager::getCurrentUserAccount();
+        $tpl = new \jTpl();
+        if ($currentUser) {
+            $tpl->assign('user', $currentUser->getInfosArray());
+        }
+        $content = $tpl->fetch('profile_index');
+        $rep->body->assign('MAIN', $content);
+    
+        return $rep;
+    }
+
+    public function modify()
+    {
+        $rep = $this->getResponse('html');
+
+        $form = jForms::create('account~profile_modify');
+
+        if (!$form) {
+            jMessage::add(jLocale::get('account.profile.modify.form.error'), 'error');
+
+            return $rep;
+        }
+
+        $tpl = new jTpl();
+        $tpl->assign('form', $form);
+        $rep->body->assign('MAIN', $tpl->fetch('profile_modify'));
+        return $rep;
+    }
+
+    public function save()
+    {
+        $rep = $this->getResponse('redirect');
+
+        $form = jForms::get('account~profile_modify');
+        
+        if (!$form) {
+            jMessage::add(jLocale::get('account.profile.modify.form.error'), 'error');
+            $rep->action = 'account~profile:index';
+
+            return $rep;
+        }
+
+        $form->initFromRequest();
+
+        if (!$form->check()) {
+            $rep->action = 'account~profile:modify';
+
+            return $rep;
+        }
+
+        $newInfos = array();
+        foreach (Account::$editableProperties as $prop) {
```

Comment:
instead of using `Account::$editableProperties` (that you should remove, see below), you could use `$form->getAllData()`.